packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -86,107 +91,114 @@ class Config:
-                    "file_name": "work.zip",
-                    "user_id": "7",
-                    "user_name": None,
-                    "file_id": "1c46752c-b096-11ea-a3c4-02420a00392e/e603724d-4af1-52a1-b866-0d4b792f8c4a/work.zip",
-                    "raw_file_path": "1c46752c-b096-11ea-a3c4-02420a00392e/e603724d-4af1-52a1-b866-0d4b792f8c4a/work.zip",
-                    "display_file_path": "Octave JupyterLab/JupyterLab Octave/work.zip",
+                    "project_name": "Octave JupyterLab",
+                },
+                # api entry (not soft link)
+                {
                     "created_at": "2020-06-17 12:28:55.705340",
+                    "entity_tag": "8711cf258714b2de5498f5a5ef48cc7b",
+                    "file_id": "api/7b6b4e3d-39ae-3559-8765-4f815a49984e/tmpf_qatpzx",
+                    "file_name": "tmpf_qatpzx",
+                    "file_size": 86,
+                    "file_uuid": "api/7b6b4e3d-39ae-3559-8765-4f815a49984e/tmpf_qatpzx",
+                    "is_soft_link": False,
                     "last_modified": "2020-06-22 13:48:13.398000+00:00",
-                    "file_size": 17866343,
-                    "parent_id": "1c46752c-b096-11ea-a3c4-02420a00392e/e603724d-4af1-52a1-b866-0d4b792f8c4a",
+                    "location_id": 0,
+                    "node_name": None,
+                    "project_name": None,
+                },
+                # api entry (soft link)
+                {
+                    "created_at": "2020-06-17 12:28:55.705340",
+                    "entity_tag": "36aa3644f526655a6f557207e4fd25b8",
+                    "file_id": "api/6f788ad9-0ad8-3d0d-9722-72f08c24a212/output_data.json",
+                    "file_name": "output_data.json",
+                    "file_size": 183,
+                    "file_uuid": "api/6f788ad9-0ad8-3d0d-9722-72f08c24a212/output_data.json",
+                    "is_soft_link": True,
+                    "last_modified": "2020-06-22 13:48:13.398000+00:00",
+                    "location_id": 0,
+                    "node_name": None,
+                    "project_name": None,
+                },
+                # datcore entry
+                {
+                    "created_at": "2020-05-28T15:48:34.386302+00:00",
+                    "entity_tag": None,
+                    "file_id": "N:package:ce145b61-7e4f-470b-a113-033653e86d3d",
+                    "file_name": "templatetemplate.json",
+                    "file_size": 238,
+                    "file_uuid": "Kember Cardiac Nerve Model/templatetemplate.json",
+                    "is_soft_link": False,
+                    "last_modified": "2020-05-28T15:48:37.507387+00:00",
+                    "location_id": 1,
+                    "node_name": None,
+                    "project_name": None,
                 },
             ]
         }
 
 
 class FileMetaDataArray(BaseModel):
-    __root__: List[FileMetaData] = []
+    __root__: List[FileMetaDataGet] = []
```

Comment:
MINOR: ``list`` not ``List``

Reply:
yes, I will fix this in the next refactoring PR step (coming soon)